corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py

```diff
@@ -18,6 +19,16 @@ class PopulateSQLCommand(BaseCommand):
     """
     AUTO_MIGRATE_ITEMS_LIMIT = 1000
 
+    @classmethod
+    def command_name(cls):
+        """
+            This needs to be implemented in each subclass even though the implementation
+            is always the same. Implementation:
+
+            return __name__.split('.')[-1]
+        """
```

Comment:
This is because `__name__` is relative to the file where it's used.  I think if you used `cls.__module__` instead, it would be the proper name.  I think.

Reply:
thank you, fixed in 890bda16dca

Reply:
:raised_hands: